flutter/realm_flutter/linux/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ include(${EPHEMERAL_DIR}/generated_config.cmake)
 execute_process(COMMAND "${FLUTTER_ROOT}/bin/flutter" "pub" "run" "realm" "install" "--target-os-type" "linux" "--package-name" "realm" #"--debug"
   OUTPUT_VARIABLE output
   RESULT_VARIABLE result
-  COMMAND_ERROR_IS_FATAL ANY
+  # COMMAND_ERROR_IS_FATAL ANY
```

Comment:
@blagoev and @fealebenpae - I commented this out to allow me to reduce the minimum required cmake version to 3.10 which is in line with the `CMakeLists` generated by `flutter create`. I think it's a bit annoying to require such a high version of cmake because it means developers have to jump through hoops to update it - e.g. on ubuntu 20.04, the highest you can get _easily_ is 3.16. Let me know if you're opposed to this change or if there are better ways to do it.

Reply:
I think we should keep the required newer cmake version. We shouldn't be sticking to what `flutter create` does since these templates are not updated regularly but we indeed require newer cmake functionality. This is the same with cocoapods. 
If we need to make it even easier for developers we could update our readme with more details how to install these.

Reply:
Is that parameter that important though? Can we replace it with something and keep the functionality with the lower cmake version?

Reply:
As far as I can tell, this only matters if we're running multiple commands which we're not.

Reply:
I have added a discussion point for this.

Reply:
Can we have a flutter plugin without this native code that compiles at a user's machine? Our native code is pre-compiled as a shared library, so these dummy plugins do nothing of value.

Reply:
This downloads the native binary though, so we definitely need it.

Reply:
I can't believe I'm saying this, but npm is better. In that case I agree with @nirinchev - it makes sense to stick to a lower CMake version, ideally the one that Flutter itself depends on, since we aren't actually compiling anything more complicated than the default plugin package template here.